pkg/agent/multicast/mcast_controller.go

```diff
@@ -222,11 +217,13 @@ type Controller struct {
 	installedGroupsMutex sync.RWMutex
 	mRouteClient         *MRouteClient
 	ovsBridgeClient      ovsconfig.OVSBridgeClient
-
 	// queryInterval is the interval to send IGMP query messages.
 	queryInterval time.Duration
 	// mcastGroupTimeout is the timeout to detect a group as stale if no IGMP report is received within the time.
-	mcastGroupTimeout time.Duration
+	mcastGroupTimeout   time.Duration
+	queryGroupId        binding.GroupIDType
+	queryGroupInstalled bool
+	queryGroupMember    map[string]uint32
```

Comment:
rename it as "queryGroupMembers"?

Reply:
Do you think `queryGroupMember` is really needed? Can we use the interfaceStore as the cache?

Reply:
OK, done